Source/Extensions/Blazorise.DataGrid/DataGrid.razor.cs

```diff
@@ -820,6 +861,79 @@ public Task Sort( DataGridColumn<TItem> column, SortDirection? sortDirection = n
         return Task.CompletedTask;
     }
 
+
+    /// <summary>
+    /// Replaces the current sorting with the given sort order
+    /// </summary>
+    /// <param name="columns">Columns used for sorting</param>
+    /// <returns>A task that represents the asynchronous operation.</returns>
+    /// <remarks>
+    /// Note that <see cref="DataGridColumn{TItem}.Sortable"/> and <see cref="Sortable"/> must be enabled to be able to sort!
+    /// If more than one column is specified, <see cref="SortMode"/> must be <see cref="DataGridSortMode.Multiple"/>
+    /// </remarks>
+    public async Task ApplySorting(params DataGridSortColumn[] columns ) {
+        if ( columns.Length > 1 && SortMode == DataGridSortMode.Single )
+        {
+            throw new ArgumentException(
+                $"Cannot sort on multiple columns when {nameof(SortMode)} is {nameof(DataGridSortMode.Single)}." );
+        }

```

Comment:
@stsrki do we like this check? 
I think we generally disregard from throwing too much exceptions in Blazorise.

Reply:
I'm generally against throwing exceptions. In this case, it could _blow_  to the user by mistake if they try to sort in "somewhat unsuported"  way,

Reply:
Fixed in c625994d8165eaffb7e33c3d6c66b72a3126b8f7